src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.html

```diff
@@ -0,0 +1,119 @@
+        <div class="btns_group">
+          <div *ngIf="order.order_status === 'formed'" class="btn_grid_cancel">
+            <button class="btn_cancel">Скасувати</button>
+          </div>
+          <div *ngIf="order.order_status === 'done_unpaid'" class="btn_grid_fake">
+            <button class="btn_fake"></button>
+          </div>
+          <div *ngIf="order.order_status === 'formed' || order.order_status === 'done_unpaid'" class="btn_grid_pay">
+            <button class="btn_pay">Оплатити</button>
+          </div>
+          <div
+            *ngIf="order.order_status === 'on_the_route' || order.order_status === 'confirmed' || order.order_status === 'done'"
+            class="btn_grid_order"
+          >
+            <button class="btn_order">Замовити ще раз</button>
+          </div>
+        </div>
+      </div>
+      <div *ngIf="order.extend" class="full_card">
+        <div class="header_details">Деталі замовлення</div>
+        <table class="table_of_details">
+          <tr class="header_table">
+            <th>Послуги</th>
+            <th>Об'єм</th>
+            <th>Вартість</th>
+            <th>Кількість пакетів</th>
+            <th>Сума</th>
+          </tr>
+          <tr class="data_table">
+            <td>Вторсировина</td>
+            <td>120 л</td>
+            <td>200 грн</td>
+            <td>1 шт</td>
+            <td>200 грн</td>
+          </tr>
+          <tr class="data_table">
+            <td>Безнадійний одяг</td>
+            <td>120 л</td>
+            <td>300 грн</td>
+            <td>1 шт</td>
+            <td>300 грн</td>
+          </tr>
+          <tr class="data_table">
+            <td>Безнадійний одяг</td>
+            <td>20 л</td>
+            <td>50 грн</td>
+            <td>2 шт</td>
+            <td>100 грн</td>
+          </tr>
+          <tr>
+            <td></td>
+          </tr>
+          <tr class="sum_of_order">
+            <td colspan="4">Сума замовлення:</td>
+            <td>500 грн</td>
+          </tr>
+          <tr *ngIf="order.order_status !== 'done_unpaid'" class="optional_row">
+            <td colspan="4">Сертифікат:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr *ngIf="order.order_status !== 'done_unpaid'" class="optional_row">
+            <td colspan="4">Бонуси:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr *ngIf="order.order_status === 'done_unpaid'" class="optional_row">
+            <td colspan="4">Сума наданих послуг:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr class="sum_to_pay">
+            <td colspan="4">Сума до оплати:</td>
+            <td>350 грн</td>
+          </tr>
+        </table>
+      </div>
+      <div *ngIf="order.extend" class="order_details">
+        <ol>
+          <li class="bold_text">Доставка з Еко магазину</li>
+          <li>Номери ваших замовлень:<span class="bold_text">&nbsp;3507777,</span>&nbsp;<span class="boldText">45678907777</span></li>
```

Comment:
We need to break this line into new lines

Reply:
Done